server/src/models/surveyResultRequest.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { SurveyResultRequestModel } from '../interfaces/surveyModel';
+
+class SurveyResultRequest {
```

Comment:
if this is an entity to be persisted within database, then could you move this class as under src/db/models . Similar comment for CosmosClient class as well.